### PR TITLE
Create bash scripts: Alex

### LINKED ISSUE #73: Write a bash script for dropping all tables and re-seeding the database, reset.sh
As a developer, I want a reset.sh script that can completely wipe and reset the database, so I can easily clear out all current data and start fresh when necessary, ensuring a clean environment for new tests or deployments

### PR DESCRIPTION
**Pull request by AlexOBarnes**
This closes #73 , #78 , #81 

What I have done:
- Created reset.sh automatically resets database and populates with seed data
- Created query.sh allows for easier querying of database
- Created connect.sh quickly connects to database
- Each of these use environment variables to automatically connect to a given database

### CODE REVIEW

File: database/connect.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+source .env
+export PGPASSWORD=$DB_PASSWORD
```
Comment (CalebMathers) [inline database/connect.sh:2]: Nice way to save having to enter the password when connecting

File: database/query.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+source .env
+export PGPASSWORD=$DB_PASSWORD
+if [ -z "$1" ]; then
+  echo "Please provide a valid SQL query as an argument."
+  exit 1
+fi
```
Comment (CalebMathers) [inline database/query.sh:6]: Bash script if statement daaamn

File: database/reset.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+source .env
+export PGPASSWORD=$DB_PASSWORD
+psql -h $DB_HOST -p $DB_PORT -U $DB_USER $DB_NAME
```
Comment (CalebMathers) [inline database/reset.sh:3]: Should this run the schema file as it is intended to reset?

**Review by CalebMathers — APPROVED** [review]

File: database/query.sh (hunk 2 above)
Comment (olliejth) [reply database/query.sh:6]: Interesting way of handling this.

**Review by olliejth — APPROVED** [review]
I think this is a smart use of bash shell scripts

File: database/connect.sh (hunk 1 above)
Comment (MurasatoNaoya) [reply database/connect.sh:2]: Saves time, otherwise the user would manually have to enter the password each time. LGTM.

**Review by MurasatoNaoya — APPROVED** [review]

File: database/reset.sh (hunk 3 above)
Comment (AlexOBarnes) [reply database/reset.sh:3]: Yes it should

**Pull request merged**